api/metrics/v1/metrics.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package clutch.metrics.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/metrics/v1;metricsv1";
+
+import "validate/validate.proto";
+import "api/v1/annotations.proto";
+import "google/api/annotations.proto";
+
+service MetricsAPI {
+  rpc GetMetrics(GetMetricsRequest) returns (GetMetricsResponse) {
+    option (google.api.http) = {
+      post : "/v1/metrics/getMetrics"
```

Comment:
is there a reason we choose `getMetrics` over query range https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries ?

Reply:
oh I see the mockups now so `getMetrics` makes sense to show the CB values

Reply:
actually I do not see this one in https://prometheus.io/docs/prometheus/latest/querying/api/